src/sentry/api/endpoints/organization_member/requests/invite/index.py

```diff
@@ -75,13 +76,14 @@ def post(self, request: Request, organization) -> Response:
         result = serializer.validated_data
 
         with transaction.atomic():
-            om = OrganizationMember.objects.create(
+            rpc_org_member = organization_service.add_organization_member(
```

Comment:
We should also likely move this outside and above this `transaction.atomic`.
Recall -- right now, it happens to be the case that these service calls are in process and can benefit from the transaction of the shared connection, but in the future, *it can't*. 
We should, as well as we can, separate work that is an API or network effect from local transactions against the resident database.